js/src/tab.js

```diff
@@ -46,10 +46,10 @@ const Tab = (() => {
     DROPDOWN              : '.dropdown',
     NAV_LIST_GROUP        : '.nav, .list-group',
     ACTIVE                : '.active',
-    ACTIVE_UL             : '> li > .active',
+    ACTIVE_UL             : ':scope > li > .active',
```

Comment:
why did you changed that selector ?

Reply:
`> li > .active` is not a valid selector. Using with `querySelector/querySelectorAll` throws a `SyntaxError`.

Reply:
yep so you should find `li` children and after that `.active` elements, no need here to implement `:scope`

Reply:
Sorry, can't understand. Are you suggesting not to use a selector? Or a double use of `.children`? In both cases it could be very slow

Reply:
Nope you're right that's better like that, sorry I didn't know `:scope` before your PR :smile:

Reply:
:scope deprecated?) https://developer.mozilla.org/en-US/docs/Web/CSS/:scope

Reply:
Nothing indicates that `:scope` selector has been deprecated. It is part of the CSS Selectors Level 4 Working Draft. Some browsers have removed  the `scoped` style tag attribute, but the `:scope` pseudo-class is still supported and works correctly in CSS (works the same as `:root`) and in `querySelector/querySelectorAll`